test/ExampleSwapToPrice.spec.ts

```diff
@@ -34,11 +34,11 @@ describe('ExampleSwapToPrice', () => {
     token0 = fixture.token0
     token1 = fixture.token1
     pair = fixture.pair
-    router = fixture.router
+    router = fixture.router01
```

Comment:
what if we just made `router` key point to latest router always? i don't expect these to fail

Reply:
agreed, done